### ISSUE TITLE
Obtrusive Title Bar on Windows 8

### ISSUE THREAD

**Issue by rittbys** [nylas/N1#1264]
Is there a way to get the Win10 title bar on Win8, or add in my own?
![capture](https://cloud.githubusercontent.com/assets/5205015/12921977/4145a7a6-cf15-11e5-96e6-19cce180232c.PNG)

**Comment by bengotow** [comment]
Hey! Thanks for filing this @rittbys — I'll see if we can get a custom titlebar in there (or just remove the frame and put the window controls icons in the bar beneath) in a future update. Stay tuned!

**Comment by rittbys** [comment]
The Discord Windows app is a good example

> On Feb 9 2016, at 12:58 pm, Ben Gotow &lt;notifications@github.com&gt;
wrote:  

>

> Hey! Thanks for filing this [@rittbys](https://github.com/rittbys) — I'll
see if we can get a custom titlebar in there (or just remove the frame and put
the window controls icons in the bar beneath) in a future update. Stay tuned!

>

> —  
Reply to this email directly or [view it on GitHub](https://github.com/nylas/N
1/issues/1264#issuecomment-182004334).![](https://github.com/notifications/bea
con/AE9sFzFQtQ-toEvIxD-TZmJbe7no0it2ks5pii44gaJpZM4HWhcn.gif)

**Comment by rittbys** [comment]
The Discord windows app is a good example.
![capture](https://cloud.githubusercontent.com/assets/5205015/12928887/b3a9abd8-cf35-11e5-9e85-6af2f8d51647.PNG)

**Comment by brittyazel** [comment]
Not to be a dick, and I know it doesn't alter the bug report, but why not upgrade from WIndows 8 to Windows 10?

**Comment by brittyazel** [comment]
@bengotow Is the option of removing using a custom titlebar and removing the frame doable under Linux as well?

**Comment by rittbys** [comment]
@brittyazel I don't want to deal with the incompatibility and privacy issues. I really hate windows 10 except for its looks and Win8.1 is working better for me.

**Comment by brittyazel** [comment]
fair enough

**Comment by rittbys** [comment]
Since Electron is a C++ application, you could probably fork Electron to have
a different window manager since I doubt it's possible to do that with JS.

> On Feb 9 2016, at 8:13 pm, Britt Yazel &lt;notifications@github.com&gt;
wrote:  

>

> fair enough

>

> —  
Reply to this email directly or [view it on GitHub](https://github.com/nylas/N
1/issues/1264#issuecomment-182169521).![](https://github.com/notifications/bea
con/AE9sF5LU-d6Rh5hNjAEA1hp5MQs4UVP9ks5pipRhgaJpZM4HWhcn.gif)

**Comment by YuriSolovyov** [comment]
I think it should be totally possible with [Frameless Window](https://github.com/atom/electron/blob/master/docs/api/frameless-window.md)

**Comment by rittbys** [comment]
@bengotow this sounds like the right way to go, I had no idea this was
actually a thing in electron.

**Comment by bengotow** [comment]
Hey folks—yep, I believe we can do this using the "Frameless Window" API. On Mac OS X, we actually do this and fake the "gumdrop" window controls in the top left. It looks like Discord is also faking the controls (they look /slightly/ different than the native ones.) I'll see what we can do.

**Comment by brittyazel** [comment]
@bengotow Is this also possible under Linux? If it were, you could unify the layouts across all platforms

**Comment by rittbys** [comment]
Yes, it's possible under Linux, I've tested it with another electron app I'm
developing.

> On Feb 19 2016, at 11:31 pm, Britt Yazel &lt;notifications@github.com&gt;
wrote:  

>

> [@bengotow](https://github.com/bengotow) Is this also possible under Linux?
If it were, you could unify the layouts across all platforms

>

> —  
Reply to this email directly or [view it on GitHub](https://github.com/nylas/N
1/issues/1264#issuecomment-186514726).![](https://github.com/notifications/bea
con/AE9sF6vLLgzz7qN3FKp2tl8yGLvV2wLuks5pl_F1gaJpZM4HWhcn.gif)

**Comment by jeffijoe** [comment]
The Windows titlebar is ruining the elegance of all the themes, so yeah, frameless window is the way to go.

**Comment by jackiehluo** [comment]
Another request in #216.

**Comment by sGerli** [comment]
I agree, this should be done in Windows to improve the design.
